src/software/thunderscope/thunderscope.py

```diff
@@ -0,0 +1,60 @@
+import os
+import signal
+
+from software.thunderscope.field.field import Field
+from software.thunderscope.log.g3log_widget import g3logWidget
+from field import obstacle_layer, path_layer, world_layer
+
+import pyqtgraph as pg
+from pyqtgraph.Qt import QtCore, QtGui
+from pyqtgraph.dockarea import *
+
+
+if __name__ == "__main__":
+
+    # Setup unix socket directory
+    try:
+        os.mkdir("/tmp/tbots")
+    except:
+        pass
+
+    # Setup MainApp and initialize DockArea
+    app = pg.mkQApp("Thunderscope")
+    app.setStyleSheet("QMainWindow{background-color: black;border: 1px solid black;}")
+    window = QtGui.QMainWindow()
+    dock_area = DockArea()
+    window.setCentralWidget(dock_area)
+    window.setWindowTitle("Thunderscope")
+
+    # Setup Field + Layers
+    field = Field()
+    field.add_layer("Vision", world_layer.WorldLayer())
+    field.add_layer("Obstacles", obstacle_layer.ObstacleLayer())
+    field.add_layer("Path", path_layer.PathLayer())
+
+    field_dock = Dock("Field", size=(500, 2000))
+    field_dock.addWidget(field)
+
+    # Setup Console Widget
+    logs = g3logWidget()
+
+    log_dock = Dock("logs", size=(500, 100))
```

Comment:
would these sizes scale nicely to different displays?

Reply:
im not sure actually, i've changed  my resolution and tried and it doesn't seem to change stuff from what i've gathered its a "size hint" maybe

@itsarune are you able to try this on your VM and see if it opens properly?

Reply:
Looks like the display is fine. I'll share a screenshot of what I see in my vm:
![sc_VirtualBox_Lubuntu 18 04_15_01_2022_11_28_37](https://user-images.githubusercontent.com/42703774/149635334-45a510cd-f25b-4b75-baaa-bd005a255e47.png)

Reply:
sweet thanks for running!, thats good, yeah then its just a size hint then, seems to scale it fine